website/content/docs/known-issues-and-solutions.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Known Issues and Solutions
+weight: 20
+group: support
+---
+
+## Background
+
+There are a couple of issues that users face while integrating or working with Netlify CMS and while the community is doing their best fixing them, here is a list of known issues and their solutions.
+
+## API Errors
+
+A list of error messages and their possible cause(s) to provide more context while debugging.
+
+#### 1. Error Message.
+
+```
+API_ERROR: {"message":"404 File Not Found"}
+```
+
+#### Possible Causes.
+
+- The collection folder or file property is missing a value (a dedicated file or folder, can't be blank or root). The file or folder should be relative to root (so `src/posts` for example).
+- There are subfolders within the collection folder.
+
+#### 2. Error Message
+
+```
+API_ERROR: Not Found
+```
+
+#### Possible Cause
+
+- The media library folder is missing, or the value of the media library property within config.yml isn't correct.
+
+#### 3. Error Message
+
+```
+Failed to load entries: API_ERROR: Not Found
+```
+
+#### Possible Causes
+
+- Git Gateway is not enabled: Enable Git Gateway from the Netlify Identity settings or refresh the token.
+- There aren't any entries for a given collection. Not sure if we should throw this error at all actually.
+
+#### 4. Error Message
+
+```
+Failed to load entry: API_ERROR: Not Found
+```
+
+#### Possible Causes
+
+- The collection folder or file property is missing a value (a dedicated file or folder, can't be blank or root). The file or folder should be relative to root (so `src/posts` for example).
+- There aren't any entries for a given collection.
+
+#### 5. Error Message
+
+```
+Failed to persist entry: API_ERROR: Not Found
+```
+
+#### Possible Cause
+
+- The org account needs to grant access to your Netlify CMS instance. [More details](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/requesting-organization-approval-for-oauth-apps).
+
+#### 6. Error Message
+
+```
+Failed to persist entry: API_ERROR: Reference update failed
+```
+
+#### Possible Cause
+
+- You're using the editorial workflow but already have a branch called cms. This error is fixed by removing the cms branch from the repo.
```

Comment:
We now handle this error and report it properly, see https://github.com/netlify/netlify-cms/pull/3879